webpack/ForemanTasks/Components/common/ClickConfirmation/index.js

```diff
@@ -67,10 +83,88 @@ ClickConfirmation.propTypes = {
   onClick: PropTypes.func.isRequired,
   confirmType: PropTypes.oneOf(['warning', 'danger']),
   id: PropTypes.string.isRequired,
+  isOpen: PropTypes.bool,
+  setModalClosed: PropTypes.func,
 };
 
 ClickConfirmation.defaultProps = {
   confirmType: 'warning',
+  isOpen: false,
+  setModalClosed: () => {},
+};
+
+// Unlock Modal Components
+const confirmationMessage = __(
+  'I understand that this may cause harm and have working database backups of all backend services.'
+);
+
+export const UnlockModal = ({ onClick, id, isOpen, setModalClosed }) => (
```

Comment:
Isn't it confusing to have two different components called `UnlockModal` defined in different places?

Reply:
Renamed to `UnlockConfirmationModal`